app/notifications/notifications.component.ts

```diff
@@ -8,12 +8,17 @@ import { NotificationService } from '../notifications/notification.service';
 @Component({
   moduleId: module.id,
   selector: 'hip-notifications',
-  templateUrl: 'notifications.component.html'
+  templateUrl: 'notifications.component.html',
+  styleUrls: ['notifications.component.css']
 })
 export class NotificationsComponent implements OnInit {
   private notifications: Notification[] = [];
   private notificationsResponseHandled = false;
   translatedResponse: any;
+  selectedStatus: string = 'all';
+  notificationStatusOptions = ['Read', 'Unread'];
+  notificationTypes = ['TOPIC_CREATED', 'TOPIC_ASSIGNED_TO', 'TOPIC_REMOVED_FROM', 'TOPIC_STATE_CHANGED','TOPIC_DEADLINE_CHANGED', 'TOPIC_DELETED', 'TOPIC_UPDATED', 'TOPIC_ATTACHMENT_ADDED' ]
```

Comment:
(max-line-length) app/notifications/notifications.component.ts[20, 1]: Exceeds maximum line length of 140
(semicolon) app/notifications/notifications.component.ts[20, 192]: Missing semicolon
(whitespace) app/notifications/notifications.component.ts[20, 106]: missing whitespace